src/components/daily/foodtooltip/FoodTooltip.js

```diff
@@ -45,8 +50,9 @@ class FoodTooltip extends React.Component {
     const food = this.props.food;
     const actualValue = this.getCarbs(food);
     const rows = [];
-    if (this.isPrescribed(food)) {
-      const prescribedValue = this.getPrescribedCarbs(food);
+    const prescriptor = this.getPrescriptor(food);
+    if (this.isPrescribed(food) && (prescriptor !== PRESCRIPTOR_NONE) ) {      
```

Comment:
do you think we need to have protection again string comparaison case (string sensitive vs non sensitive value) ?

Reply:
not needed as value is managed by platform data.